### ISSUE TITLE
The metrics of YOLO v8-seg becomes zero during training.

### ISSUE THREAD

**Issue by yingchaoAo** [ultralytics/ultralytics#9088]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

![WX20240319-120601@2x](https://github.com/ultralytics/ultralytics/assets/145567458/7a5fbfd2-e48d-4525-9858-8b2960f708bc)
At the same time, the metrics of Box also begin to fall.

### Environment

NVIDIA RTX 4090. 

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@yingchaoAo hey there! 👋 It seems like you're encountering an issue where the metrics for YOLOv8-seg drop to zero during training. This can sometimes happen due to various reasons, including learning rate settings, dataset anomalies, or even a temporary glitch in the training process.

A quick thing to try is adjusting your learning rate and monitoring the training process closely to see if the metrics improve. Here's a small snippet on how you could slightly modify the learning rate if using the CLI:

```bash
yolo detect train data=<your_dataset>.yaml model=yolov8n-seg.pt lr0=0.001
```

And for Python API:

```python
from ultralytics import YOLO

model = YOLO('yolov8n-seg.pt')
model.train(data='<your_dataset>.yaml', lr0=0.001)
```

Make sure to replace `<your_dataset>.yaml` with your actual dataset configuration file. If this doesn't help, there might be a need to review your dataset for anomalies or even consider experimenting with different model configurations.

Let us know if tweaking the learning rate helps or if you're still facing issues. We’re here to help out! 🛠️
